news/_posts/2020-03-11-improvements-on-contributors-board.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title:  "Improvements to come on contributors board"
+subtitle: "More features, more modern, and more information about contributors"
+date:  2020-03-12 15:30:00
+authors: [AntoineThomas]
+icon: icon-board
+image: /assets/images/theme/meta-logo-build.png
+tags: [contribute, community]
+---
+
+
+After all those years without any modification nor improvements, it is really time to think about the future of the "[Top contribution achievements](http://contributors.prestashop.com/)" board. While discussing with many people from the community on how to open the governance of the open source project, and how to involve more contributors, new needs have been identified. This article is about quick wins, upcoming developments, and about the general spirit behind this work.
+
+## But, why?
+
+In 2020, a lot of work is going to be done around the PrestaShop open source project to make it more attractive for contributors. It must of course take into account that most people don't really contribute out of generosity, because at the end of the month, everybody has to pay their bills. So, like with other well-known open source projects, people and companies who take time to contribute to PrestaShop must be able to show off: being visible in the community as an expert is valuable to attract customers, it provides proof that they know what they are doing with their PrestaShop business.
```

Comment:
```suggestion
A lot of work will be done in 2020 around the PrestaShop open source project to make it more attractive for contributors. It must, of course, take into account that most people don't really contribute out of generosity because, at the end of the month, everybody has to pay their bills. So, like in other open source projects, people and companies who take time to contribute to PrestaShop must be able to show off: being visible in the community as experts is valuable to attract customers, it provides proof that they know what they are doing with their PrestaShop business.
```